s3/stow_test.go

```diff
@@ -61,6 +61,9 @@ func TestPreSignedURL(t *testing.T) {
 	is.NoErr(err)
 
 	container, err := location.Container("flyte-demo")
+	if err != nil {
+		t.Skip(err)
+	}
 	ctx := context.Background()
 	res, err := container.PreSignRequest(ctx, stow.ClientMethodPut, "blah/bloh/fileon", stow.PresignRequestParams{
```

Comment:
Response generated during testing
```
https://[bucket].s3.us-west-2.amazonaws.com/blah/bloh/fileon?X-Amz-Algorithm=[alg]&X-Amz-Credential=[cred]&X-Amz-Date=20240126T221416Z&X-Amz-Expires=3600&X-Amz-Security-Token=[token]&X-Amz-SignedHeaders=host%3Bx-amz-server-side-encryption%3Bx-amz-server-side-encryption-aws-kms-key-id&X-Amz-Signature=[sig]
```
Note the section `X-Amz-SignedHeaders=host%3Bx-amz-server-side-encryption%3Bx-amz-server-side-encryption-aws-kms-key-id`